jobs/job_calculate_precision.py

```diff
@@ -139,14 +141,21 @@ def check_fr(fr_list):
 
     return 0.5
 
-def check_rational(p, q):
+def check_rational(p, q, num):
```

Comment:
Maybe we should add an explanation on this function since it's trying to check for rationality in many ways